src/modules/shared/components/PhoneCard/PhoneCard.tsx

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import cn from 'classnames';
+import style from './PhoneCard.module.scss';
+import { PhoneItem } from '../../../../types/PhoneItem';
+// eslint-disable-next-line max-len
+import phoneImage from '../../../../static/img/phones/apple-iphone-7/black/00.jpg';
+import { useAppSelector } from '../../../../store/hooks';
+
+type Props = {
+  phoneItem: PhoneItem;
+};
+
+export const PhoneCard: React.FC<Props> = ({ phoneItem }) => {
+  const {
+    name,
+    fullPrice,
+    price,
+    screen,
+    capacity,
+    ram,
+    // image,
+  } = phoneItem;
+
+  const { isDarkTheme } = useAppSelector((state) => state.theme);
+  const isItemSelected = true;
+
+  const showDiscountPrice = () => {
+    if (price !== fullPrice) {
+      return (
+        <p
+          className={cn(style.productCard__fullPrice, {
+            [style.productCard__fullPrice__DARK]: isDarkTheme,
+          })}
+        >
+          {`$${fullPrice}`}
+        </p>
+      );
+    }
+
+    return false;
+  };
```

Comment:
You don't need to do this, just do it through conditional rendering in jsx
```suggestion
      {price !== fullPrice && (
        <p
          className={cn(style.productCard__fullPrice, {
            [style.productCard__fullPrice__DARK]: isDarkTheme,
          })}
        >
          {`$${fullPrice}`}
        </p>
      )}

Reply:
I don't this siggestion is better because of how this conditional rendering works. If we have false condition it will return 0 and we will see it on the page. That't why I implemented function